docs/src/pages/index.astro

```diff
@@ -6,13 +6,16 @@ import Layout from '../layouts/MainLayout.astro';
   // This is some wip redirect code based on the browser language.
   // A vercel.json redirect is enforced in production, so no user should ever see this page.
   // Remove the vercel.json redirect when this is ready.
-  const KNOWN_LANGUAGES = ['en', 'nl', 'fi','zh-CN','zh-TW'];
-  let newLang = (window.navigator.userLanguage || window.navigator.language || 'en').substr(0, 2);
+  const KNOWN_LANGUAGES = ['de','en','es','fi','nl','pt-br','zh-CN','zh-TW'];
```

Comment:
Would it be possible to construct a list of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl objects here instead? So we can have better verification that we're using valid codes + hopefully `newLangWithRegion` and `newLang` can just be function calls..

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/Locale a list of these

Reply:
Yes! I'm not super familiar with this API but if you / someone wants to tackle in a follow up PR I'd love to review it!